packages/kotti-ui/source/kotti-field-file-upload/components/TakePhoto/utils/web-rtc.ts

```diff
@@ -3,11 +3,10 @@
  * @returns true if supported, else false
  */
 export const isWebRTCSupported = (): boolean =>
-	!(
-		!window.navigator.mediaDevices ||
-		!window.navigator.mediaDevices.enumerateDevices ||
-		!window.navigator.mediaDevices.getUserMedia
-	)
+	// eslint-disable-next-line @typescript-eslint/unbound-method
```

Comment:
This will crash if `navigator.mediaDevices` does not exist, it should either be wrapped in a `try`/`catch` or the existence of `mediaDevices` needs to be checked again